assets/js/util/convert-time.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Time related utility functions.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Converts seconds into an array with the following values [HH, MM, SS, MS] as described below.
+ * HH: hours.
+ * MM: minutes.
+ * SS: seconds.
+ * MS: milliseconds.
+ *
+ * For example, passing 196.385 returns [ 0, 3, 16, 385 ].
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} seconds The number of seconds.
+ * @return {Array} Array containing the hours, minutes, seconds and milliseconds.
+ *
+ */
+export const convertSecondsToArray = ( seconds ) => {
+	seconds = parseFloat( seconds );
+
+	if ( isNaN( seconds ) || 0 === seconds ) {
+		return [ 0, 0, 0, 0 ];
+	}
+
+	return [
+		Math.floor( seconds / 60 / 60 ),
+		Math.floor( ( seconds / 60 ) % 60 ),
+		Math.floor( seconds % 60 ),
+		parseInt( parseFloat( seconds ).toFixed( 3 ).slice( -3 ) ),
```

Comment:
Rather than parsing this back and fourth and using string operations, we can get the decimal value as an integer up to the thousandths precision with some basic math like so:
```suggestion
		Math.floor( seconds * 1000 ) - Math.floor( seconds ) * 1000,
```

For example: if `seconds` were `123.4567`, this would result in the following:
```
123456 - 123000 = 456
```

Reply:
nice. updated!